src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString
+    .trim()
+    .split(';')
+    .filter((style) => style.trim() !== '');
+
+  const stylesObject = styles.reduce((acc, style) => {
+    const [property, value] = style.split(':', 2)
+      .map((item) => item.trim());
+
+    if (property.length > 0 && value.length > 0) {
+      acc[property] = value;
```

Comment:
It's good that you're checking for the length of the property and value before adding them to the accumulator object. This prevents adding invalid entries.